spec/support/schemas/v1/users/registrations/create/create.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
It's not right json schema. Please use latest [specification](http://json-schema.org/draft/2019-09/release-notes.html) or generator.

Reply:
Schemas for user registration 201 and 422 error added.